testdata/singledel_manual_compaction_set_with_del

```diff
@@ -174,10 +174,8 @@ compact a-b L2
 ----
 1:
   000004:[a#10,SINGLEDEL-a#10,SINGLEDEL]
-3:
-  000008:[a#9,SET-a#8,DEL]
 4:
-  000007:[a#7,SET-a#7,SET]
+  000008:[a#9,SET-a#9,SET]
```

Comment:
@jbowens while cleaning up the ML PR, I noticed the need change the results in this test case. Are ML compactions violating some invariant here?  Note that this test still fails on some later line.